controllers/state_of_the_world.go

```diff
@@ -130,16 +132,40 @@ func NewPolicyMachineryController(manager ctrlruntime.Manager, client *dynamic.D
 		// TODO: add tls policy specific tasks to workflow
 	}
 
+	ok, err = openshift.IsConsolePluginInstalled(manager.GetRESTMapper())
+	if err != nil || !ok {
+		logger.Info("console plugin is not installed, skipping related watches and reconcilers", "err", err)
+	} else {
+		controllerOpts = append(controllerOpts,
+			controller.WithRunnable("consoleplugin watcher", controller.Watch(&consolev1.ConsolePlugin{}, openshift.ConsolePluginsResource, metav1.NamespaceAll)),
```

Comment:
Should we use label predicates here?

```suggestion
			controller.WithRunnable("consoleplugin watcher", controller.Watch(&consolev1.ConsolePlugin{}, openshift.ConsolePluginsResource, metav1.NamespaceAll, controller.FilterResourcesByLabel[*consolev1.ConsolePlugin](fmt.Sprintf("app=%s", consoleplugin.KUADRANT_CONSOLE)))),
```

Reply:
done